sdks/python/apache_beam/io/gcp/tests/bigquery_matcher.py

```diff
@@ -192,7 +193,10 @@ def _get_query_result(self, bigquery_client):
       if len(response) >= len(self.expected_data):
         return response
       time.sleep(1)
-    raise TimeoutError('Timeout exceeded for matcher.')
+    if six.PY3:
```

Comment:
We've decided against using `six` in the Beam codebase. You can check for Python version using this code instead:
```suggestion
    if sys.version_info >= (3,):
```

Reply:
Thanks @udim.
I'll make the change.